api/v1/server/handlers/api-tokens/revoke.go

```diff
@@ -24,5 +25,8 @@ func (a *APITokenService) ApiTokenUpdateRevoke(ctx echo.Context, request gen.Api
 		return nil, err
 	}
 
+	ctx.Set(string(constants.ResourceIdKey), apiToken.ID.String())
+	ctx.Set(string(constants.ResourceTypeKey), string(constants.ResourceTypeApiToken))
```

Comment:
```suggestion
	ctx.Set(string(constants.ResourceTypeKey), constants.ResourceTypeApiToken.String())
```

thought that was the point of adding the string method?